src/bundle/Resources/public/js/scripts/embedded.item.actions.js

```diff
@@ -0,0 +1,308 @@
+(function (global, doc, ibexa, Routing, Translator, Popper) {
+    const MENU_PROPS = {
+        placement: 'bottom-end',
+        fallbackPlacements: ['bottom-start', 'top-end', 'top-start'],
+    };
+    const token = document.querySelector('meta[name="CSRF-Token"]').content;
+    const siteaccess = document.querySelector('meta[name="SiteAccess"]').content;
+    const metaLanguageCode = document.querySelector('meta[name="LanguageCode"]')?.content;
+    const previewLanguageCode = metaLanguageCode ?? ibexa.adminUiConfig.languages.priority[0];
+    const adminUiLanguages = ibexa.adminUiConfig.languages.mappings;
+    const emdedItemsUpdateChannel = new BroadcastChannel('ibexa-emded-item-live-update');
+    const editEmbeddedItemForm = doc.querySelector('[name="embedded_item_edit"]');
+    const actionsMenuTriggerBtns = doc.querySelectorAll('.ibexa-embedded-item-actions__menu-trigger-btn');
+    const updateNode = ({ node, value, isMiddleEllipsis }) => {
+        if (!isMiddleEllipsis) {
+            node.textContent = value;
+
+            return;
+        }
+
+        const middleEllipsisNode = node.querySelector('.ibexa-middle-ellipsis');
+        const middleEllipsisNameStartNode = node.querySelector(
+            '.ibexa-middle-ellipsis__name--start .ibexa-middle-ellipsis__name-ellipsized',
+        );
+        const middleEllipsisNameEndNode = node.querySelector('.ibexa-middle-ellipsis__name--end .ibexa-middle-ellipsis__name-ellipsized');
+
+        middleEllipsisNode.title = value;
+        middleEllipsisNameStartNode.textContent = value;
+        middleEllipsisNameEndNode.textContent = value;
+
+        ibexa.helpers.ellipsis.middle.parse(node);
```

Comment:
We have `ibexa.helpers.ellipsis.middle.update` , as far as I understand that's what you wanted to achieve here